CMakeLists.txt

```
@@ -2494,7 +2494,7 @@ if(NOT CURL_DISABLE_INSTALL)
       COMMAND ${CMAKE_COMMAND} -P "${CMAKE_CURRENT_BINARY_DIR}/CMake/cmake_uninstall.cmake")
   endif()
 
-  install(FILES "${PROJECT_SOURCE_DIR}/scripts/mk-ca-bundle.pl"
+  install(FILES "${PROJECT_SOURCE_DIR}/scripts/wcurl"
```

```suggestion
  install(FILES "${PROJECT_SOURCE_DIR}/scripts/mk-ca-bundle.pl" "${PROJECT_SOURCE_DIR}/scripts/wcurl"
```
Let's also keep `mk-ca-bundle.pl`.

---

Why? It is not installed by the autotools version and I don't think distros and people in general ship it anywhere. I don't think it was ever intended to be installed.

---

It was unclear to me if this was by accident or intentional. No objection to not install it anymore though.

---

Hm, on a second thought I'm still confused because then why build and distribute the manpage for `mk-ca-bundle.pl`? I feel a separate PR would be a better place to deal with this.

---

This seems fine indeed. In 5023ffad2c27d4b916ddb91800f99ecc5d3aad07 #13197 I overlooked something and thought autotools is installing `mk-ca-bundle.pl`, while it isn't.

But, CMake should probably not install `mk-ca-bundle.1` either, which it currently does.

---

To not install the man page, this can work:
```diff
--- a/docs/CMakeLists.txt
+++ b/docs/CMakeLists.txt
@@ -39,7 +39,7 @@ if(BUILD_MISC_DOCS)
       VERBATIM
     )
     add_custom_target("curl-generate-${_man_misc}.1" ALL DEPENDS "${_man_target}")
-    if(NOT CURL_DISABLE_INSTALL)
+    if(NOT CURL_DISABLE_INSTALL AND NOT _man_misc STREQUAL "mk-ca-bundle")
       install(FILES "${_man_target}" DESTINATION "${CMAKE_INSTALL_MANDIR}/man1")
     endif()
   endforeach()
```